### PR TITLE
Add a resource calculation widget component

### PR DESCRIPTION
**Pull request by p-netm**
part of #960.

Adds a component for resource calculation

### CODE REVIEW

**Comment by p-netm** [comment]
Currently:
This only contains the component only, it has not been hooked up with the other Jurisdiction Assignment code, This will happen once #1065 is merged.

File: src/containers/pages/JurisdictionAssignment/helpers/ResourceCalcWidget/tests/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      `"1 daysat a rate of   structures per team per day with  Teams"`
+    );
+
+    // backspace teams so that we have no data, value would be ""
+    wrapper
+      .find('input[name="teamsCount"]')
+      .simulate('change', { target: { value: '', name: 'teamsCount' } });
+    wrapper.update();
+
+    // see if there has been any change to the number of days, should start with 1 days
```
Comment (p-netm) [inline src/containers/pages/JurisdictionAssignment/helpers/ResourceCalcWidget/tests/index.test.tsx:87]: this should read `should start with 0 days`

**Comment by p-netm** [comment]
Screenshot from 2020-08-12 16-58-20

File: src/containers/pages/JurisdictionAssignment/helpers/ResourceCalcWidget/tests/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+      'structures count input'
+    );
+    wrapper
+      .find('input[name="teamsCount"]')
+      .simulate('change', { target: { value: 3, name: 'teamsCount' } });
+    wrapper.update();
+
+    // see if there has been any change to the number of days, should start with 1 days
+    expect(wrapper.find('form p').text()).toMatchInlineSnapshot(
+      `"1 daysat a rate of   structures per team per day with  Teams"`
```
Comment (moshthepitt) [inline src/containers/pages/JurisdictionAssignment/helpers/ResourceCalcWidget/tests/index.test.tsx:101]: Is it possible to show the structures and teams fields changing with different input in tests?

Comment (p-netm) [reply src/containers/pages/JurisdictionAssignment/helpers/ResourceCalcWidget/tests/index.test.tsx:101]: yeah, sure.

**Pull request merged**